src/scheduler/helpers.ts

```diff
@@ -0,0 +1,62 @@
+import {
+  Audit,
+  ListAnnualReviewPostReminderType,
+  ListEventJsonData,
+  ListItemAnnualReviewProviderReminderType,
+} from "server/models/types";
+import { logger } from "server/services/logger";
+import { ListItemWithHistory } from "server/components/dashboard/listsItems/types";
+import { isLocalHost, SERVICE_DOMAIN } from "server/config";
+
+const defaultTodayDateString = new Date().toLocaleString("default", { year: "numeric", month: "long", day: "numeric" });
+export const now = new Date(Date.now());
+const todayDateString = new Date(Date.UTC(now.getFullYear(), now.getMonth(), now.getDate()));
+
+export function parseDate(dateString = defaultTodayDateString) {
+  const date = new Date(Date.parse(dateString));
+  return new Date(Date.UTC(date.getFullYear(), date.getMonth(), date.getDate()));
+}
+
+export function formatDate(date: Date = todayDateString) {
+  const options: Intl.DateTimeFormatOptions = { day: "2-digit", month: "short", year: "numeric" };
+  return date.toLocaleString('en-gb', options);
+}
+
```

Comment:
mentioned above but just use iso. `parseDate` and `formatDate` would be made redundant if so. then you can just do `new Date(isoString)` if you need access to the date object/methods.

Reply:
Just one thing to be aware of is when running the process manually for a specific date, the date format yyyy-MM-dd will need to be used.